ed_sensor_integration/tools/fitter_viz.cpp

```diff
@@ -183,6 +183,53 @@ double getYaw(const geo::Mat3& rotation)
     return angles.yaw;
 }
 
+void drawLine(
+        cv::Mat& canvas, geo::Vec2 point1, geo::Vec2 point2, geo::Transform2 pose, float resolution, int origin_x, int origin_y, cv::Scalar color)
+{
+    // computing outer corner positions (in meters)
+    geo::Vec2 tf_point1 = pose * point1;
+    geo::Vec2 tf_point2 = pose * point2;
```

Comment:
`tf_point` lets me think. That it is a `tf::Point` or similar. So please change naming.

Reply:
changed to rel_point for relative position